.circleci/config.yml

```
@@ -916,6 +919,36 @@ jobs:
       - store_test_results:
           path: test/test-results/e2e
 
+  test-e2e-chrome-multiple-providers:
+    executor: node-browsers
+    parallelism: 1
+    steps:
+      - run: *shallow-git-clone
+      - attach_workspace:
+          at: .
+      - run:
+          name: Move test build to dist
+          command: mv ./dist-test ./dist
+      - run:
+          name: Move test zips to builds
+          command: mv ./builds-test ./builds
+      - run:
+          name: Activate Selenium Manager for Chrome
+          command: node_modules/selenium-webdriver/bin/linux/selenium-manager --browser chrome
+      - run:
+          name: test:e2e:chrome:multi-provider
+          command: |
+            if .circleci/scripts/test-run-e2e.sh
+            then
+              yarn test:e2e:chrome:multi-provider --retries 2
+            fi
+          no_output_timeout: 20m
```

This change was made after you started your PR
```suggestion
          no_output_timeout: 5m
```